backend/go.mod

```diff
@@ -76,7 +76,7 @@ require (
 	github.com/jinzhu/now v1.1.2 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
-	github.com/labstack/echo/v4 v4.6.3 // indirect
```

Comment:
it is an indirect dep, if you run go mod tidy it will be reverted, no?

Reply:
No :(

Reply:
Noooooooo